### ISSUE TITLE
Salinidade, Temperatura e Altura de Onda

### ISSUE THREAD

**Issue by juoceano** [LaPoGeoMar/Proj_Modelagem_Oleo#5]
@ocefpaf precisamos de um valor de temperatura e salinidade e um campo de altura de onda para cada um dos cenários (verão inverno) para a @francielleholtz inserir no modelo.

**Comment by juoceano** [comment]
@francielleholtz confirma para mim se a altura precisa de um valor ou de um campo de valores...

**Comment by francielleholtz** [comment]
> 

Apenas um valor seria suficiente.

**Comment by ocefpaf** [comment]
No notebook https://github.com/LaPoGeoMar/Proj_Modelagem_Oleo/blob/main/itajai/itajai-buoy.ipynb a gente viu que o _deployment_ da boia mais próximo dos poços é o 5. Esse  o _deployment_ tem dados que cobre aproximadamente o ano de 2017 inteiro. Em https://github.com/LaPoGeoMar/Proj_Modelagem_Oleo/pull/7 coloquei um notebook que limpa os dados da boia e separa apenas o que bate com os dados do HyCOM.

**Comment by francielleholtz** [comment]
@ocefpaf, abri o "extract_points_deployment_5.ipynb". Para complementar, vi que tem velocidade e direção do vento- que são dois parâmetros que preciso também (não sabia até recentemente). 
Só para confirmar, e me desculpa se estou errada, mas nesse notebook não especifica a média dos valores, correto? Se for o caso, teria como compilar os resultados do 5 em uma média para valores de temperatura da água, salinidade, altura de ondas, velocidade do vento e direção do vento? Não sei se a melhor opção seria pegar todo o período disponível no notebook ou fazer a média entre 2016-2017, que será o período usado. Você e a @juoceano sabem dizer qual o melhor modo de ser feita essa média?
Obrigada!

**Comment by ocefpaf** [comment]
> Para complementar, vi que tem velocidade e direção do vento- que são dois parâmetros que preciso também (não sabia até recentemente).

Precisa deles em um ponto ou na grade?

> Só para confirmar, e me desculpa se estou errada, mas nesse notebook não especifica a média dos valores, correto? 

Não. Como fazer as médias vai depender de cada caso/simulação que você criar.

> Você e a @juoceano sabem dizer qual o melhor modo de ser feita essa média?

Posso te ajudar a fazer a média mas, vamos dizer, que um dos casos é a simulação para a primavera, devemos fazer a mesma média para a primavera como nos dados do HyCOM.

Vamos fazer assim, faz uma tabela com os dados que precisa para cada simulação, mais ou menos assim:

|              | Temperature | vel-water   | vel-wind   |
|-----------|-------------------|---------------|---------------|
| Grade   |                      | &#10004;  |                  |
| Ponto   | &#10004;      |                   | &#10004; |

E uma outra com as simulações que pretende fazer, tipo:
- [ ] sazonal com o ano 2017
- [ ] mês-a-mês para Jan, Feb, etc
- [ ] Semana X onde ocorreu passagem da frente Y


Com a tabela e a lista das simulações ficará mais fácil preparar os dados.
Não precisa ter tudo que pretende fazer ali, mas coloque alguma pra gente começar.

**Comment by francielleholtz** [comment]
certo, entendi! amanhã terei uma conversa rápida com a Ju sobre isso. após, mando a tabela, okay? muito obrigada!

**Comment by juoceano** [comment]
A princípio - por questão do tempo de entrega do TCC - vamos fazer somente cenário de verão e inverno médios daquele ano. Para a parte do modelo que calculo quanto óleo emulsifica, sedimenta, evapora etc precisa de dados pontuais para temperatura, salinidade, altura de onda, direção e intensidade do vento - um para cenário de inverno e um para cenário de verão. 

A @francielleholtz está organizando a tabela e em breve irá enviá-la.

**Comment by francielleholtz** [comment]
[Dados - Página1.pdf](https://github.com/LaPoGeoMar/Proj_Modelagem_Oleo/files/9880420/Dados.-.Pagina1.pdf)
Segue a tabela. É bem simples: todos os valores são pontuais, e são necessárias as médias para duas simulações (verão e inverno médio).
Pelo que entendi da disponibilidade, o verão seria na passagem de 2016/2017 e inverno 2017.

**Comment by ocefpaf** [comment]
@francielleholtz vou colocar aqui em markdown para ficar mais fácil de visualizar. Por favor checa se tá tudo OK antes do proseguirmos:

|              | Water temperature | wave height   | salinity | wind speed    | direction-wind | water velocity |
|--------------|-------------------|---------------|----------|---------------|----------------|----------------|
| Point        | &#10004;          | &#10004;      | &#10004; | &#10004;      | &#10004;       |                |
| Grid         |                   |               |          |               |                | &#10004;       |

Simulações:
- [ ] verão médias de dez-2016, jan-2017, mar-2017
- [ ] inverno médias de ago-2017, set-2017, oct-2017

**Comment by francielleholtz** [comment]
Boa noite. @ocefpaf até onde eu sei, não é necessário water velocity. Seriam apenas: temperatura, altura de onda, salinity, velocidade e direção do vento.

**Comment by francielleholtz** [comment]
As simulações são essas mesmas,

**Comment by francielleholtz** [comment]
boa tarde a todos! @ocefpaf tens uma previsão sobre quando as médias ficam prontas? pergunto apenas para me organizar. muito obrigada!

**Comment by ocefpaf** [comment]
Para salinity vamos usar 35 por enquanto, estou providenciando os dados do HyCOM de salinidade. Os campos de velocidade e médias no ponto da boia eu coloco aqui assim que terminar o notebook.

| season   |   water_temperature |   wave_significant_height |    u_wind |   v_wind |
|:---------|--------------------:|--------------------------:|----------:|---------:|
| DJF      |             26.3226 |                   1.54966 |  1.08057  |  2.94717 |
| JJA      |             21.7435 |                   1.91505 | -1.11985  |  2.30075 |
| MAM      |             25.6204 |                   1.93701 |  0.725439 |  1.79078 |
| SON      |             22.3104 |                   2.07684 |  0.20074  |  2.51841 |

**Comment by francielleholtz** [comment]
não precisa do campo de velocidade. essas são as variáveis necessárias. muito obrigada

**Comment by francielleholtz** [comment]
será que seria necessário esperar as salinidades ou podemos usar 35 como padrão- o gnome mesmo fornece esse valor de padrão. o que acha? @ocefpaf @juoceano

**Comment by francielleholtz** [comment]
estava rodando no pygnome com o arquivo do Hycom já disponível aqui no git @ocefpaf . e aparentemente rodou normal, então acredito que o campo das velocidades não seria necessário.

**Comment by ocefpaf** [comment]
> será que seria necessário esperar as salinidades ou podemos usar 35 como padrão- o gnome mesmo fornece esse valor de padrão. o que acha? @ocefpaf @juoceano

35 tá OK. Até vou pegar as salinidade para o futuro apenas por "completeness" mas vai mudar pouco seus resultados.

**Comment by ocefpaf** [comment]
> estava rodando no pygnome com o arquivo do Hycom já disponível aqui no git @ocefpaf . e aparentemente rodou normal, então acredito que o campo das velocidades não seria necessário.

Não me lembro de deixar lá as médias sazonais. Creio que tem o ano todo. Mas já coloco um para cada estação.

**Comment by francielleholtz** [comment]
@ocefpaf pode me explicar melhor? eu baixei o arquivo de todo o período e estou rodando com ele. vc adicionou os períodos de cada estação, mas posso continuar rodando com o arquivo completo? tem necessidade de eu baixar esses? ademais, tem previsão da salinidade? estou com uma certa pressa e já tinha rodado considerando o 35.

**Comment by francielleholtz** [comment]
@ocefpaf entendi! eu baixo o arquivo com as médias no mesmo local que anteriormente você havia me mostrado para baixar o arquivo anterior? vi que o hycom-oil.nc foi atualizado ontem, seria esse ou o hycom_season_avg.nc?

**Comment by francielleholtz** [comment]
ademais, @ocefpaf , quando você acredita que a salinidade ficaria pronta? pensei em usar 35 por ser até um valor default no gnome, mas, se vai mudar os resultados, e você acredita que é possível entregar essas médias, acho que seria interessante ter esses valores.

**Comment by ocefpaf** [comment]
> vi que o hycom-oil.nc foi atualizado ontem, seria esse ou o hycom_season_avg.nc?

O segundo. Esse notebook tem um exemplo e como ler o arqquivo: https://nbviewer.org/github/LaPoGeoMar/Proj_Modelagem_Oleo/blob/main/03-read-seasons.ipynb


> ademais, @ocefpaf , quando você acredita que a salinidade ficaria pronta? pensei em usar 35 por ser até um valor default no gnome, mas, se vai mudar os resultados, e você acredita que é possível entregar essas médias, acho que seria interessante ter esses valores.

Vai levar mais um tempinho. Mas duvido muito que vai mudar alguma coisa, seria só por "correctness" mesmo. Para você se convencer disso pega os limites climatológicos da salinidade na região e faz uma simulação com o mínimo e o máximo esperado e vê o que muda.
